### PR TITLE
Components feature

### PR DESCRIPTION
**Pull request by ulisesedu7**
In this project, I've implemented the Calculator Component Feature styles in React using Class-Based Components and rendering the calculator component into the main app.js file.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,9 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Calculator />
   );
```
Comment (Whoistolu) [inline src/App.js:7]: - [ ] [REQUIRED] Kindly ensure that the `App.js`  component is class-based in order to properly follow the project requirements.

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @ulisesedu7 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._
- [ ] [REQUIRED] In order to have a more professional project, kindly ensure you remove the create react app automated README and update it with the Microverse professional README.

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (ulisesedu7) [reply src/App.js:7]: You're right @Whoistolu! Thank you so much for the reminder, I totally missed that, I'll start fixing it.

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @ulisesedu7 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Great job making the required changes and converting the App.js into a class-based component that imports the Calculator component
- Good communication with the previous reviewer

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ulisesedu7** [comment]
@MahmoudBakr23 Thanks!!

**Pull request merged**